VERSION

```diff
@@ -1 +1 @@
-1.0.1
+1.0.2
```

Comment:
Thank you for adjusting the version! 🙂 One minor thing - Hera uses [semantic versioning](https://github.com/argoproj-labs/hera-workflows/blob/main/CONTRIBUTING.md#contributing-to-hera). This indicates a PATCH, which implies a bug fix, but this is a feature! So, I suggest we bump the MINOR version, to something like `1.1.0` perhaps.